src/pages/somyi/Login/LoginSomyi.js

```diff
@@ -1,7 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './LoginSomyi.scss';
+
 const LoginSomyi = () => {
-  return <h1>Somyi Hello</h1>;
+  const [saveIdValue, setSaveIdValue] = useState('');
+  const [savePasswordValue, setSavePasswordValuee] = useState('');
+  const isValidation =
+    saveIdValue.indexOf('@') > -1 && savePasswordValue.length >= 5;
+
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate('/main-somyi');
+  };
+
+  const saveUserId = e => {
+    setSaveIdValue(e.target.value);
+  };
+
+  const saveUserPassword = e => {
+    setSavePasswordValuee(e.target.value);
+  };
+
+  return (
+    <div className="loginSomyi">
+      <h1>Westagram</h1>
+      <form>
+        <input
+          type="text"
+          placeholder="전화번호, 사용자 이름 또는 이메일"
+          value={saveIdValue}
+          onChange={saveUserId}
+        />
+        <input
+          type="password"
+          placeholder="비밀번호"
+          value={savePasswordValue}
+          onChange={saveUserPassword}
+        />
+        <button
+          type="button"
+          className={isValidation ? 'disabled' : 'albled'}
+          disabled={isValidation ? false : true}
```

Comment:
```suggestion
          disabled={!isValidation}
```
isValidation 값이 그 자체로 boolean 값이기 때문에 이와 같이 수정할 수 있겠습니다!